config/app.default.php

```diff
@@ -236,7 +236,7 @@
             'timezone' => 'UTC',
             'flags' => [],
             'cacheMetadata' => true,
-            'log' => false,
+            'log' => true,
```

Comment:
This is not a good default to have. People will leave this on by accident and soak up all their disk.